### DISCUSSION TITLE
Vídeo 01 - Orientações Gerais

### DISCUSSION THREAD

**Discussion by thpoiani** [thpoiani/lab-quarkus#1] [General]
Olá e seja bem-vindo! É ótimo ter você aqui depois de assistir ao **Vídeo 01 - Orientações Gerais** da **Aula - Desenvolvendo um Sistema para Eleição Usando Quarkus Framework** distribuído pela [DIO](https://dio.me/).
Espero que você tenha gostado e que tenha aprendido algo novo. Este fórum é um espaço para discussões e compartilhamento de conhecimentos, ideias e experiências entre desenvolvedores como nós.
Sinta-se à vontade para participar das discussões e fazer perguntas. Juntos, podemos criar uma comunidade de aprendizado e colaboração.

**Comment by thpoiani** [comment]
O que você acha de comentar um pouco sobre o que achou da primeira aula? Aproveite e deixe seu LinkedIn aqui.
Considere me adicionar: https://www.linkedin.com/in/thpoiani/

**Reply by DiegoMouraOficial** [reply]
Excelente aula! Tenho conhecimento básico de Java e comecei a estudar o Quarkus para aproveitar uma oportunidade de trabalhar em uma squad, onde a empresa (banco) planeja migrar o sistema legado (arquitetura monolítica) para microservices utilizando o Quarkus. É um grande desafio, mas pretendo aprofundar meus conhecimentos com o auxílio das suas aulas.

**Reply by jeoliveira94** [reply]
A primeira aula foi muito boa; o professor explicou de forma clara como será o curso, quais ferramentas serão usadas e a finalidade de cada uma delas. A escolha de oferecer leituras mais aprofundadas dos conceitos como uma opção é excelente. Isso aproveita bem o recurso da videoaula, onde o aluno pode pausar e ler com calma, se achar necessário, sem que o professor precise gastar tempo lendo em aula. Iniciei este curso principalmente por causa do Quarkus, e fiquei bastante surpreso com as diversas tecnologias apresentadas. Não conheço algumas delas, mas espero adquirir muito conhecimento até o final. Grande abraço!

**Reply by didifive** [reply]
Deixou claro o que o título disse, orientações gerais e expectativas.
Gostei da abordagem com a apresentação das ferramentas, é uma ótima ideia, trabalhou muito bem a questão de curso assíncrono, para quem já conhece uma ferramenta é só passar, para quem não conhece, dá um pause e lê. O vídeo não fica pesado com teoria, quem já conhece a ferramenta não precisa de pular o vídeo ou esperar toda a leitura e ainda deixa uma base legal sobre as ferramentas para quem não conhece ainda.

**Reply by ItaloBaciliere** [reply]
No final de 2023, concluí meu curso de Ciência da Computação na FUMEC. Meu trabalho de conclusão de curso envolveu uma análise comparativa dos frameworks Spring Boot, Micronaut e Quarkus. Ao compará-los, cheguei à conclusão de que o Quarkus está inovando significativamente no desenvolvimento de aplicações Java, especialmente em ambientes de grande escalabilidade.

Isso me mostrou que devo me atualizar sobre o Quarkus o quanto antes. Será extremamente importa para o desenvolvimento da minha carreira.

**Reply by ale-fuentes-ar** [reply]
Boa noite @thpoiani , grato por compartir seu conhecimento.

Sou um dev que utiliza java, e estou ampliando conhecimentos das diferentes ferramentas como Quarkus, Spring, etc.

Sobre a primeira aula, fiquei empolgado pela quantidade de recursos o tools mencionados, somente vi o 1er video disponibillizado, no se se está o curso completo, tomara que sim, para poder ter uma inmerso mas geral, completa e de já de qualidade.

LinkedIn : https://www.linkedin.com/in/raul-alejandro-fuentes/

**Reply by jonasmzsouza** [reply]
Excelente aula! 
A abordagem  e gama de ferramentas que serão utilizadas mostra o quão enriquecedor será o conhecimento adquirido ao longo do curso. 
Ansioso para absorver o máximo possível e agregar na minha carreira como desenvolvedor.

**Comment by siqueira76** [comment]
Caramba @thpoiani, essa apresentação foi um super "esquenta", a propriedade que vc demonstrou em apresentar cada tópico e tecnologias, foi bem impressionante!

**Comment by joseeasantos** [comment]
Parabéns pelo conteúdo @thpoiani! Esse realmente foi um curso de qualidade! Entendo que o tempo foi curto para a quantidade de assuntos abordados, mas deu um bom embasamento para pesquisas e extensões. 
Assuntos como mensageria de forma simples, programação reativa, arquitetura, padrões e práticas de programação, o próprio uso do redis, métricas do(s) sistema(s), idéias sobre coisas novas lançadas em Java como Loom foram de grande relevância para mim e creio que para grande parte dos alunos que viram e verão esse curso.
Meu [LinkedIn](https://www.linkedin.com/in/jos%C3%A9-espedito-albernaz-dos-santos-232a9a8b/)

**Comment by vicssb** [comment]
Parabéns @thpoiani !!!
Explanação muito clara sobre o tema.
Já trabalho com Java a muitos anos e estava querendo começar com o Quarkus e o treinamento "caiu no meu colo de presente" 😜
Vamos com toda a empolgação para o projeto!

**Comment by mauromortoni** [comment]
Inicialmente, a capacidade da mente parece limitada diante da abundância de informações complexas e variadas.
Com a prática os desafios nos faz buscar soluções e fixar o conhecimento.

**Comment by jonasfschuh** [comment]
Boa tarde Thiago @thpoiani, gostei demais da qualidade da apresentação e da primeira aula. Parabéns pelo seu emprego no Canadá também. Como feedback, de todos os cursos que fiz aqui na DIO, esse de longe foi o mais interessante de forma técnica, que apresenta todo arsenal em tempo real. 
Como sugestão, se pensar em fazer outro curso, iria recomendar fazer um site de apostas com resiliência pra atuar em nível nacional, de alta performance. Algo como um "paredão do bigbrother" que necessita alta demanda. 
OBS: Já te dei add no git.

**Comment by marceloscientist** [comment]
Opa, bora lá! Ansioso para o que vamos desbravar aqui! Bem completo o conteúdo e interessante...

**Comment by GiancarloVieira** [comment]
Primeira aula já promete! Sua didática é muito boa e eficiente, Thiago. Essa dica de deixar as faixas de parar o vídeo para ler o slide foi genial, todo curso eu fico na dúvida se ouço ou leio o slide do professor kkkk Mas assim, parando e lendo, fica melhor e aumenta o nível de absorção do conteúdo. Valeu
